aspnetcore-webapi/RestVsWebSocket/Client/Program.cs

```diff
@@ -0,0 +1,30 @@
+﻿// See https://aka.ms/new-console-template for more information
+using System.Net.WebSockets;
+using System.Text;
+
+var ws = new ClientWebSocket();
+
+await ws.ConnectAsync(new Uri("ws://localhost:5289/ws"),
+    CancellationToken.None);
+
+Console.WriteLine("Connected!");
+
+var recieveTask = Task.Run(async () =>
```

Comment:
As stated earlier on the API project, this should be called `receiveTask` instead of `recieveTask`